internal/servers/controller/handlers/targets/target_service.go

```diff
@@ -565,6 +568,120 @@ func (s Service) RemoveTargetCredentialLibraries(ctx context.Context, req *pbs.R
 	return &pbs.RemoveTargetCredentialLibrariesResponse{Item: item}, nil
 }
 
+// AddTargetCredentialSources implements the interface pbs.TargetServiceServer.
```

Comment:
Just a reminder to add unit tests for these new function.

Reply:
Done